pages/common/git-var.md

```diff
@@ -0,0 +1,12 @@
+# git var
+
+> Prints a Git logical variable's value.
+> More information: <https://git-scm.com/docs/git-var>.
+
+- Print the value of a Git logical variable:
+
+`git var {{GIT_AUTHOR_IDENT|GIT_COMMITTER_IDENT|GIT_EDITOR|GIT_PAGER}}`
+
+- List all Git logical variables (including those found in `.git/config`; deprecated in favor of `git config -l`):
+
+`git var -l`
```

Comment:
Why include a deprecated command?

Reply:
It's not deprecated, just the listing of the config variables is deprecated.  If you run `git var -l`, you'll notice the same output as `git config -l`, plus some `CONSTANT_CASE` variables at the end.  This command is still in Git and still works.

Reply:
Oh, I see. So only half the command's functionality is deprecated?

Reply:
Yes.  Exactly.

Reply:
Should we remove the example @sbrl ?

Reply:
@sbrl @navarroaxel take a look at https://github.com/tldr-pages/tldr/commit/b680abe810d86b37a9cdbba0223cc2f2b2bac00d